dev/com.ibm.websphere.appserver.features/visibility/public/jpa-3.0/com.ibm.websphere.appserver.jpa-3.0.feature

```diff
@@ -0,0 +1,234 @@
+ org.eclipse.persistence.platform.database.converters; type="third-party", \
+ org.eclipse.persistence.platform.database.events; type="third-party", \
+ org.eclipse.persistence.platform.database.jdbc; type="third-party", \
+ org.eclipse.persistence.platform.database.oracle.annotations; type="third-party", \
+ org.eclipse.persistence.platform.database.oracle.jdbc; type="third-party", \
+ org.eclipse.persistence.platform.database.oracle.plsql; type="third-party", \
+ org.eclipse.persistence.platform.database.partitioning; type="third-party", \
+ org.eclipse.persistence.platform.database; type="third-party", \
+ org.eclipse.persistence.platform.server; type="third-party", \
+ org.eclipse.persistence.platform.xml.jaxp; type="third-party", \
+ org.eclipse.persistence.platform.xml; type="third-party", \
+ org.eclipse.persistence.sequencing; type="third-party", \
+ org.eclipse.persistence.services.websphere; type="third-party", \
+ org.eclipse.persistence.services; type="third-party", \
+ org.eclipse.persistence.sessions.broker; type="third-party", \
+ org.eclipse.persistence.sessions.changesets; type="third-party", \
+ org.eclipse.persistence.sessions.coordination.broadcast; type="third-party", \
+ org.eclipse.persistence.sessions.coordination.corba.sun; type="third-party", \
+ org.eclipse.persistence.sessions.coordination.corba; type="third-party", \
+ org.eclipse.persistence.sessions.coordination.jms; type="third-party", \
+ org.eclipse.persistence.sessions.coordination.rmi; type="third-party", \
+ org.eclipse.persistence.sessions.coordination; type="third-party", \
+ org.eclipse.persistence.sessions.factories; type="third-party", \
+ org.eclipse.persistence.sessions.interceptors; type="third-party", \
+ org.eclipse.persistence.sessions.remote.corba.sun; type="third-party", \
+ org.eclipse.persistence.sessions.remote.rmi.iiop; type="third-party", \
+ org.eclipse.persistence.sessions.remote.rmi; type="third-party", \
+ org.eclipse.persistence.sessions.remote; type="third-party", \
+ org.eclipse.persistence.sessions.serializers; type="third-party", \
+ org.eclipse.persistence.sessions.server; type="third-party", \
+ org.eclipse.persistence.tools.file; type="third-party", \
+ org.eclipse.persistence.tools.profiler; type="third-party", \
+ org.eclipse.persistence.tools.schemaframework; type="third-party", \
+ org.eclipse.persistence.tools.tuning; type="third-party", \
+ org.eclipse.persistence.tools.weaving.jpa; type="third-party", \
+ org.eclipse.persistence.tools; type="third-party", \
+ org.eclipse.persistence.transaction.was; type="third-party", \
+ org.eclipse.persistence.transaction; type="third-party", \
+ org.eclipse.persistence; type="third-party"
+IBM-ShortName: jpa-3.0
+Subsystem-Name: Jakarta Persistence API 3.0
+-features=com.ibm.websphere.appserver.transaction-1.2, \
+ com.ibm.websphere.appserver.jdbc-4.2; ibm.tolerates:="4.3", \
+ com.ibm.websphere.appserver.jpaContainer-3.0,\
+ com.ibm.websphere.appserver.org.eclipse.persistence-2.7
```

Comment:
should this be 3.0 instead of 2.7?

Reply:
yeah it should.  Probably worked in my sandbox because I had 2.7 enabled for enhancement, and didn't clean everything out after cloning it for 3.0.